Sources/eppo/ConfigurationStore.swift

```diff
@@ -2,11 +2,20 @@ import Foundation
 
 class ConfigurationStore {
     private var configuration: Configuration?
-    
     private let syncQueue = DispatchQueue(
         label: "com.eppo.configurationStoreQueue", attributes: .concurrent)
     
+    private let fileURL: URL
+    
+    // Initialize with the disk-based path for storage
     public init() {
+        // Set the file path for storing the configuration
+        let fileManager = FileManager.default
+        let urls = fileManager.urls(for: .documentDirectory, in: .userDomainMask)
+        self.fileURL = urls[0].appendingPathComponent("configuration.json")
+        
+        // Load any existing configuration from disk when initializing
+        self.configuration = loadFromDisk()
```

Comment:
minor: would it make sense to spawn this concurrently, so the store is returned immediately and configuration is populated shortly after?